src/elliptic/curves/ed25519.rs

```diff
@@ -483,8 +483,9 @@ impl Serialize for Ed25519Point {
     {
         let bytes = self.pk_to_key_slice();
         let bytes_as_bn = BigInt::from(&bytes[..]);
+        let padded_bytes_hex = format!("{:0>64}", bytes_as_bn.to_hex());
```

Comment:
1) Can you explain what this line is doing ? 
2) why 64 is hard coded?

Reply:
1. this line pads the string to 64 hex characters (i.e. length of a compressed Ed25519 public key, https://tools.ietf.org/html/rfc8032#section-3.1.
2. No explicit reason, I can use a constant if you see need.